plugins/tracker-resources/src/index.ts

```diff
@@ -152,28 +152,31 @@ export { default as AssigneeEditor } from './components/issues/AssigneeEditor.sv
 
 export { CreateProject, IssuePresenter, TitlePresenter }
 
+const DEFAULT_LIMIT = 200
 export async function queryIssue<D extends Issue> (
```

Comment:
I think all of this will be removed after we switch to new search?

Reply:
Yes, it's true! And I could move it out from this PR too if we would like just accept the part with elastic changes

Reply:
We should remove and switch to new API,

Also you need to continue work on search API, since right now it is pure elastic, but it should be our minimal APIs suitable for our search. And in case we could like to replace elastic in future we will be able to do so with just adapter replacement.